pkg/plugins/resources/postgres/migrations/data/1589041445_add_unique_id_and_owner.up.sql

```diff
@@ -0,0 +1,35 @@
+ALTER TABLE resources
+    DROP CONSTRAINT resources_pkey,
+    ADD CONSTRAINT resources_pkey PRIMARY KEY (name, mesh, type);
+ALTER TABLE resources
+    DROP COLUMN namespace;
+ALTER TABLE resources
+    ADD COLUMN owner_name varchar(100);
+ALTER TABLE resources
+    ADD COLUMN owner_mesh varchar(100);
+ALTER TABLE resources
+    ADD COLUMN owner_type varchar(100);
+ALTER TABLE resources
+    ADD CONSTRAINT owner_fk FOREIGN KEY (owner_name, owner_mesh, owner_type) REFERENCES resources (name, mesh, type) ON DELETE CASCADE;
+
+DELETE
+FROM resources
+WHERE type = 'DataplaneInsight';
+
+-- set owner for all resources except Mesh
+UPDATE resources r1
+SET owner_name = r2.name,
```

Comment:
shouldn't this be `SET owner_name = r2.mesh`? I expect we set an owner for all resources to be the mesh. For example a TrafficPermission with mesh `default` has an owner of Mesh `default`

Reply:
In r2's row we will have mesh according to there WHERE condition: `r2.name = r1.mesh AND r2.type = 'Mesh' AND r1.type != 'Mesh'`
So doesn't matter which attribute to choose: mesh or name, they are the same for Mesh record.